lib/common.sh

```diff
@@ -330,11 +330,7 @@ export KUBERNETES_BINARIES_VERSION="${KUBERNETES_BINARIES_VERSION:-${KUBERNETES_
 export KUBERNETES_BINARIES_CONFIG_VERSION=${KUBERNETES_BINARIES_CONFIG_VERSION:-"v0.15.1"}
 
 # Ephemeral Cluster
-if [[ "${CONTAINER_RUNTIME}" = "docker" ]]; then
-  export EPHEMERAL_CLUSTER=${EPHEMERAL_CLUSTER:-"kind"}
-else
-  export EPHEMERAL_CLUSTER="minikube"
-fi
+export EPHEMERAL_CLUSTER="minikube"
```

Comment:
```suggestion
export EPHEMERAL_CLUSTER=${EPHEMERAL_CLUSTER:-"minikube"}
```

Reply:
ow yeah, this I did intentionally here for the test because it gets dictated from different places like project-infra. CAPM3 etc